filebeat/module/mysql/slowlog/_meta/fields.yml

```diff
@@ -108,6 +129,34 @@
       description: >
         Slow log rate limit, a value of 100 means that one in a hundred queries
         or sessions are being logged.
+    - name: read_first
```

Comment:
Should we make this `read.first` etc?

Reply:
This module is keeping the names of the original fields, I'd continue using these values.